transports/quic/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "libp2p-quic"
-version = "0.7.0-alpha"
+version = "0.7.0-alpha.0"
```

Comment:
It is released without the `0` so I think we can't just change that now?

Reply:
Hmm, would `cargo publish` reject this version? the next version is `-alpha.2`, does it have a check to ensure the prerelease number is incremental?

Reply:
What is the upside of publishing a new version?

Reply:
We need a version to depend on from libp2p 0.50 that has tls pinned, so that it wont pull alpha.2 anymore

Reply:
Ah yes, you are completely right. I forgot about the dependency from `libp2p-quic` to `libp2p-tls`.